ci/github/run_asan_clang.sh

```
@@ -0,0 +1,26 @@
+export CLANG_VER=`clang --version | perl -lne 'print $1 if /version (\d+)/'`
+echo "Clang major version is: $CLANG_VER"
+# Enable undefined behaviour sanitizer using traps
+CFLAGS_USAN="-g3 -O0 -mtune=generic -fno-omit-frame-pointer -fsanitize=address -fno-sanitize-recover"
+LDFLAGS_STD="-fsanitize=address -Wl,-Bsymbolic-functions -Wl,-z,relro"
+
+# Sanitizer options to continue avoid stopping the runs on leaks (expected on postgres binaries)
+export ASAN_OPTIONS=halt_on_error=false,leak_check_at_exit=false,exitcode=0
+export MSAN_OPTIONS=halt_on_error=false,leak_check_at_exit=false,exitcode=0
+
+#Run postgres preloading sanitizer libs
+LD_PRELOAD=/usr/lib/clang/${CLANG_VER}/lib/linux/libclang_rt.asan-x86_64.so
```

Shellcheck (0.10.0)

[warning] LD_PRELOAD appears unused. Verify use (or export if used externally).

(SC2034)